packages/fxa-react/lib/test-utils/index.ts

```diff
@@ -81,42 +80,31 @@ async function getFtlFromPackage(packageName: PackageName, locale: string) {
  * locales pull from the cloned l10n repo in `public`.
  */
 export async function getFtlBundle(packageName: PackageName, locale = 'en') {
-  const messages = await getFtlFromPackage(packageName, locale);
+  const path = getFtlPath(packageName, locale);
+  const messages = await fs.promises.readFile(path, 'utf-8');
+  return _createFtlBundle(messages, locale);
+}
+
+/**
+ * Get the specified FTL file/bundle synchronously. Note, this should be avoided. Using the asynchronous version, getFtlBundle, is preferred.
+ * @param packageName Which package to load the bundle for
+ * @param locale Which locale FTL bundle to load. 'en' will load `test/[name].ftl` and other
+ * locales pull from the cloned l10n repo in `public`.
+ * @returns
+ */
+export function getFtlBundleSync(packageName: PackageName, locale = 'en') {
+  const path = getFtlPath(packageName, locale);
+  const messages = fs.readFileSync(path, 'utf-8');
+  return _createFtlBundle(messages, locale);
+}
+
+function _createFtlBundle(messages: string, locale: string) {
   const resource = new FluentResource(messages);
   const bundle = new FluentBundle(locale, { useIsolating: false });
   bundle.addResource(resource);
   return bundle;
 }
 
-function testMessage(
```

Comment:
Moved to utils.tsx. We now use these checks to validate calls state prior to calling l10n.getString.